environment/src/main/java/com/sequenceiq/environment/credential/service/CredentialService.java

```diff
@@ -176,6 +176,7 @@ public Credential create(Credential credential, @Nonnull String accountId, @Nonn
         credential.setResourceCrn(credentialCrn);
         credential.setCreator(creatorUserCrn);
         credential.setAccountId(accountId);
+        grpcUmsClient.assignResourceOwnerRoleIfEntitled(creatorUserCrn, credentialCrn, accountId);
```

Comment:
please put this into the transaction block

Reply:
okay